hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -171,7 +172,8 @@ public final class Hedera implements SwirldMain {
      */
     private PlatformStatus platformStatus = PlatformStatus.STARTING_UP;
 
-    private HandleThrottleAccumulator handleThrottling;
+    private ThrottleAccumulator throttleAccumulator;
+    private ThrottleAccumulator synchronizedThrottleAccumulator;
```

Comment:
Any better ideas for naming this field? Since we have a `SynchronizedThrottleAccumulator` class and it's a bit confusing. Maybe `throttleAccumulatorWithCapacitySplit`?

Reply:
We could use `hapiThrottling` and `handleThrottling` maybe.

Reply:
Or `backendThrottle` and `frontendThrottle`. If we use these terms consistently throughout the code base, they will become project language.